gpflow/kernels.py

```diff
@@ -683,42 +683,75 @@ def Kdiag(self, X, presliced=False):
 
 class Periodic(Kernel):
     """
-    The periodic kernel. Defined in  Equation (47) of
+    The periodic family of kernels. Can be used to wrap any Stationary kernel
+    to transform it into a periodic version. The canonical form (based on the
+    SquaredExponential kernel) can be found in Equation (47) of
 
     D.J.C.MacKay. Introduction to Gaussian processes. In C.M.Bishop, editor,
     Neural Networks and Machine Learning, pages 133--165. Springer, 1998.
 
-    Derived using an RBF kernel once mapped the original inputs through
-    the mapping u=(cos(x), sin(x)).
+    The derivation can be achieved by mapping the original inputs through the
+    transformation u = (cos(x), sin(x)).
 
-    The resulting kernel can be expressed as:
-        k(r) =  σ² exp{ -0.5 sin²(π r / γ) / ℓ²}
+    For the SquaredExponential base kernel, the result can be expressed as:
+        k(r) = σ² exp{ -0.5 sin²(π r / γ) / ℓ² }
 
     where:
-    r  is the Euclidean distance between the input points
+    r is the Euclidean distance between the input points,
     ℓ is the lengthscale parameter,
     σ² is the variance parameter,
     γ is the period parameter.
 
-    (note that usually we have a factor of 4 instead of 0.5 in front but this is absorbed into lengthscale
-    hyperparameter).
+    (note that usually we have a factor of 4 instead of 0.5 in front but this
+    is absorbed into lengthscale hyperparameter).
     """
+    def __init__(self, input_dim=None, period=1.0, variance=1.0,
+                 lengthscales=1.0, basekern=None, active_dims=None, name=None):
+        """
+        The Periodic kernel supports the specification of any Stationary kernel
+        as the base through the `basekern` parameter. If this is provided then
+        `input_dim`, `variance`, `lengthscales` and `active_dims` parameters are
+        taken from there.
+
+        If `basekern` is not provided, then a SquaredExponential is constructed
+        implicitly using the relevant arguments:
+
+            basekern = SquaredExponential(
```

Comment:
the following isn't quite right (it doesn't pass the period, it does pass the variance), but you could also just say "is constructed implicitly using the input_dim, variance, lengthscales, active_dims arguments".

Reply:
Oops, will fix!